core/src/query.rs

```diff
@@ -199,7 +189,7 @@ pub fn create_archive_index(name: CheckedName<'_>) -> Result<String, PgmqError>
 pub fn create_index(name: CheckedName<'_>) -> Result<String, PgmqError> {
     Ok(format!(
         "
-        CREATE INDEX IF NOT EXISTS msg_id_vt_idx_{name} ON {PGMQ_SCHEMA}.{TABLE_PREFIX}_{name} (vt ASC, msg_id ASC);
+        CREATE INDEX IF NOT EXISTS {TABLE_PREFIX}_{name}_vt_idx ON {PGMQ_SCHEMA}.{TABLE_PREFIX}_{name} (vt ASC);
```

Comment:
there will be index on `msg_id` implicitly due to primary key change in the table DDL, correct?